CoroutineTemplate/domain/src/main/java/co/nimblehq/domain/usecase/UseCaseResult.kt

```diff
@@ -1,4 +1,4 @@
-package co.nimblehq.coroutine.domain.usecase
```

Comment:
FYI: If we update the package name here, the script to init project will not work 😄 

https://github.com/nimblehq/android-templates/blob/c301ab05aca6249cbe8e62351a5ef35e0e1b7aad/newproject.sh#L71-L82

Reply:
@hoangnguyen92dn Good catch, I've updated the package name.